intro.Rmd

```diff
@@ -32,7 +25,7 @@ The workflow around an R package can also improve
 the tractability and reproducibility for an analysis project [@marwick2018packaging].
```

Comment:
```suggestion
the tractability and reproducibility for an analysis project [@marwick2018packaging].

We will revisit folder structure when we discuss project management for a clinical trail project.  
```

Reply:
I believe the correct word is traceability?